Rakefile

```diff
@@ -143,7 +143,7 @@ task 'gem:native', [:plat] do |t, args|
   verbose = ENV['V'] || '0'
 
   grpc_config = ENV['GRPC_CONFIG'] || 'opt'
-  ruby_cc_versions = ['3.3.0', '3.2.0', '3.1.0', '3.0.0'].join(':')
+  ruby_cc_versions = ['3.4.0', '3.3.5', '3.2.0', '3.1.0', '3.0.0'].join(':')
```

Comment:
Changing to `3.3.5` is necessary due to the notes at https://github.com/rake-compiler/rake-compiler-dock/releases/tag/v1.6.0 (Ruby `3.3.0` had an issue with cross-compilation only fixed in  `3.3.5`)